.jenkins/baroclinic_initialization.sh

```diff
@@ -63,7 +63,7 @@ ${JENKINS_DIR}/install_virtualenv.sh ${VIRTUALENV}
 source ${VIRTUALENV}/bin/activate
 
 CMD="srun python3 ${PACE_DIR}/driver/examples/baroclinic_init.py ${JENKINS_DIR}/driver_configs/${experiment}.yaml"
-run_command "${CMD}" Job${action} ${scheduler_script}
+run_command "${CMD}" Job${action} ${scheduler_script} $minutes
```

Comment:
Thanks for also updating this file. We may in the near future decide to delete this file since this isn't running on CI now.